modules/git/tree_entry.go

```diff
@@ -108,6 +108,11 @@ func (te *TreeEntry) IsRegular() bool {
 	return te.gogitTreeEntry.Mode == filemode.Regular
 }
 
+// IsExecutable if the entry is an executable file (not necessarily binary)
+func (te *TreeEntry) IsExecutable() bool {
+	return te.gogitTreeEntry.Mode == filemode.Executable
```

Comment:
This isn't in the original PR. Please submit a PR to master with this change first, thenit can be backported

Reply:
Sorry: I'm on that.